tools/text_normalization/data/months.tsv

```diff
@@ -0,0 +1,58 @@
+Jan.	January
+Feb.	February
+Mar.	March
```

Comment:
These .tsv files are in the `data` folder but it looks more like `utils` or `helpers`

Reply:
I think its ok to leave it as `data` since it is hardcoded data